README.md

```diff
@@ -1886,6 +1886,10 @@ The following extractors use this feature:
 #### nflplusreplay
 * `type`: Type(s) of game replays to extract. Valid types are: `full_game`, `full_game_spanish`, `condensed_game` and `all_22`. You can use `all` to extract all available replay types, which is the default
 
+#### ADN
+* `language`: Which language version to extract. Valid values are `fr` and `de`. If not provided, the language is extracted from the provided URL
+* `season`: When downloading a show, download only the specified season. If not provided, all seasons are downloaded
```

Comment:
Just return the `season_number` in the video metadata and user can do `--match-filter`. `--extractor-args` are meant for use-cases that are otherwise unachievable

Reply:
Didn't know this was possible to be honest, then this is in deed not needed